proto/backend.proto

```diff
@@ -97,6 +97,7 @@ message CallResource {
     int32 code = 1;
     map<string,StringList> headers = 2;
     bytes body = 3;
+    bool done = 4;
```

Comment:
alternatively, this could be a `keepAlive` or `continue` flag that must be set to true

Reply:
After my investigations returning from grpc method in server should automatically result in a `EOF` at client so the flag should not be needed.

Reply:
:+1: Sweet! I was hoping we could solve it this way instead of introducing a flag/param.